streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBTimeOrderedKeyValueBuffer.java

```diff
@@ -45,17 +53,24 @@ public class RocksDBTimeOrderedKeyValueBuffer<K, V> extends WrappedStateStore<Ro
     private Serde<V> valueSerde;
     private final String topic;
     private int seqnum;
+    private final boolean loggingEnabled;
+    private final Map<Bytes, BufferValue> dirtyKeys = new HashMap<>();
+    private int partition;
+    private String changelogTopic;
+    private InternalProcessorContext context;
 
     public RocksDBTimeOrderedKeyValueBuffer(final RocksDBTimeOrderedKeyValueBytesStore store,
                                             final Duration gracePeriod,
-                                            final String topic) {
+                                            final String topic,
+                                            final boolean loggingEnabled) {
         super(store);
         this.gracePeriod = gracePeriod.toMillis();
         minTimestamp = Long.MAX_VALUE;
         numRecords = 0;
         bufferSize = 0;
         seqnum = 0;
         this.topic = topic;
+        this.loggingEnabled = true;
```

Comment:
This line sets the instance variable  `loggingEnabled` to `true`, ignoring the `loggingEnabled` parameter on line 65 - is this intentional?